kmod/src/dir.c

```diff
@@ -1770,7 +1812,10 @@ const struct file_operations scoutfs_dir_fops = {
 	.llseek		= generic_file_llseek,
 };
 
-const struct inode_operations scoutfs_dir_iops = {
+
+
+const struct inode_operations_wrapper scoutfs_dir_iops = {
+	.ops = {
```

Comment:
Ooof, *shakes fist at kabi*.  I guess we'll just do this and deal with the breakage and compat as we move forward to mainline.  Please describe this in the commit message.

Reply:
want a separate commit for this?

Reply:
Nah, just describe in the commit message that we had to add nested structs for the rhel kabi expansion and that we'll run into compat pain when we get it building in mainline.